docs/use_cases/fine_grained_authorization/intro.md

```diff
@@ -0,0 +1,180 @@
+# Use Case: Fine-Grained Authorization
+
+This use case demonstrates how to build a high-performance, real-time **policy engine** using Feldera.
+A policy engine enforces access control **rules** that determine which actions users can perform on various
+resources, and under which conditions.
+We focus on a family of access control models called **Fine-Grained-Authorization** (FGA) models, which were
+first popularized by Google's [Zanzibar](https://research.google/pubs/zanzibar-googles-consistent-global-authorization-system/)
+system.
+
+**We demonstrate how Feldera's best-in-class support for incremental computation and mutually recursive SQL queries
+enables users to implement complex authorization policies with only a few lines of SQL, and to
+enforce them efficiently at scale.**
```

Comment:
technically enforcement is outside feldera.
we only can answer authorization queries